tools/gcp-org-hierarchy-viewer/gcpohv_cli.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+import collections
+import functools
+
+import asciitree
+from google.auth import exceptions
+from google.oauth2 import service_account
+from googleapiclient import discovery, errors
+
+_ORG = '🏢'
+_FOLDER = '📁'
+_PROJ = '📦'
+
+
```

Comment:
I know it's a small script, but could you add a small header for each function (except main, maybe)?
Something like: 

function name - high-level description
input parameters, if any
outputs,  if any

This makes it easier to dive into the code, without reading the whole script